docs/maintenance-plan.md

```diff
@@ -0,0 +1,189 @@
+# Maintenance plan
+
+This document outlines every piece of the platform that need upgrading and
+updating.
+
+## Lagoon
+
+Update interval: 1-4 months
```

Comment:
Instead of just an interval, I would like a planned time of update, e.g. "start of every quarter", so we are more precise with when we do things.

Alternatively, a schedule at the end that links to the other sections, so e.g. "Every quarter: Lagoon, ...")

Reply:
i would agree with you, but at the moment we haven't been able to make that kind of promise to ourselves. This, I feel, is more precise as it states intention